lib/sundials-config/prepare.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
```

Comment:
Please add documentation to this script. At the very least, let the reader know the doc is in a markdown, but I'd much rather there be documentation here also.

Here's how I think you can make this much more useful (I'll put in inline comments):
1. Start by `git rm` the existing sundials library.
2. `git commit` after line 13.
3. `git commit` after line 18.